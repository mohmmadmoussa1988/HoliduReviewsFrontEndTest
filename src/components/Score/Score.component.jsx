import React from 'react';
import { ScoreWrapper, CurrentScore, TotalScore } from './Score.styles';

const Score = ({ children }) => {
    return (
        <ScoreWrapper>
            <CurrentScore>{children}</CurrentScore>/<TotalScore>5</TotalScore>
        </ScoreWrapper>
    )
}

export default Score;