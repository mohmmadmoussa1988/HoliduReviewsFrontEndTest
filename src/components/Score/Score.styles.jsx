import styled from 'styled-components';
export const ScoreWrapper = styled.div`
background-color:#0276DB;
border-radius:8px;
padding:5px 10px;
margin-right:14px;
display:flex;
flex-direction: row;
color: #fff;
font-size: 16px;
padding: 4.5px 10px;
line-height: 23px;
`;

export const CurrentScore = styled.div`
font-family: 'SF-bold';
padding:0 3px 0 0;

`;

export const TotalScore = styled.div`
font-family: 'SF-regular';
padding:0 0px 0px 3px;

`;

