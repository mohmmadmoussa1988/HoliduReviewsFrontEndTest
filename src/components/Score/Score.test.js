import React from "react";
import { configure, shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Score from "./Score.component";
describe(`<Score />`, () => {
  it("should render Score", () => {
    const wrapper = shallow(<Score></Score>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
