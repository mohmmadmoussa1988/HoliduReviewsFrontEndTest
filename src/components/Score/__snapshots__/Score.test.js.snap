// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Score /> should render Score 1`] = `
<styled.div>
  <styled.div />
  /
  <styled.div>
    5
  </styled.div>
</styled.div>
`;
